src/log_event.jl

```diff
@@ -45,17 +46,17 @@ function log_events(log_group::AbstractString, log_stream::AbstractString)
     # We've hit the end of the stream if the next token matches the current one.
     while next_token != curr_token || next_token === nothing
         response = try
-            @mock get_log_events(
-                logGroupName=log_group,
-                logStreamName=log_stream,
-                nextToken=next_token,
+            @mock Cloudwatch_Logs.get_log_events(
+                log_group, log_stream,
+                Dict("nextToken"=>next_token);
+                aws_config=aws_config,
```

Comment:
I *think* this fits in the line length

```suggestion
            @mock Cloudwatch_Logs.get_log_events(
                log_group, log_stream, Dict("nextToken"=>next_token);
                aws_config=aws_config,
```